### PR TITLE
Assignment 2 Submission

### PR DESCRIPTION
**Pull request by mzkhan87**
I'm submitting 4 files. 
1. print_grid.py        It prints the grid for exercise 1. 
2. FizzBuzz.py          Exercise 2. 
3. Series.py              This file contains the methods that return the nth term of the fibonacci series, lucas 
                                series. 
4. series_upto_n.py optional file where fibonacci, lucas series are returned as a string up to the nth term. 

Please let me know if I should improve the docStrring in my methods or any other kind of feedback help me improve my programming skills.

### CODE REVIEW

**Comment by mzkhan87** [comment]
I've submitted the assignment #2 for the review. Please provide me the feedback. Thanks.

File: students/MzKhan/lesson02/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Date: 02/12/2019
+    Assignment02
+
+'''
+
+# The method uses the mod division to print the desired message.
+# if number is the multiple of 3, it prints Fizz.
+# if number is the multiple of 5, it prints Buzz.
+# if number is the multiple of 3 AND 5, it prints FizzBuzz.
+def FizzBizz():
```
Comment (charlesrobison) [inline students/MzKhan/lesson02/FizzBuzz.py]: Function names in python should be lower case, words separated by underscores.

File: students/MzKhan/lesson02/FizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        elif msg % 3 == 0:
+            msg = 'Fizz'
+        elif msg % 5 ==0:
+            msg = 'Buzz'
+        print(msg)
+
+# The main method to test the code.
+if __name__ == "__main__":
+
+    FizzBizz()
```
Comment (charlesrobison) [inline students/MzKhan/lesson02/FizzBuzz.py]: Works well

File: students/MzKhan/lesson02/print_grid.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+if __name__ == '__main__':
+    print('Part 1')
+    print_default_grid()
+    print('Part 2')
+    print_grid(3)
+    print_grid(15)
+    print('Part 3')
+    print_grid2(3,4)
+    print_grid2(5,3)
+    print_grid2(6,5)
```
Comment (charlesrobison) [inline students/MzKhan/lesson02/print_grid.py]: Looks great!

File: students/MzKhan/lesson02/series.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    print('The n = {}th term of the Fibonacci series: {}'.format(n,fibonacci(n)))
+    print()
+    print('The n = {}th term of the Lucas series: {}'.format(n,lucas(n)))
+    print()
+    n=15
+    print('The n = {}th term of the Lucas series: {}'.format(n,sum_series(n,second=1,first=2)))
+    print()
+    print('The n = {}th term of the Fibonacci series: {}'.format(n,sum_series(n,second=1,first=0)))
+    print()
+    print('The n = {}th term of the Other series: {}'.format(n,sum_series(n,second=2,first=2)))
```
Comment (charlesrobison) [inline students/MzKhan/lesson02/series.py]: Good job here - nice assert statements

**Pull request merged**